packages/react/src/components/Dialog/index.tsx

```diff
@@ -170,8 +170,8 @@ export default class Dialog extends React.Component<DialogProps, DialogState> {
   }
 
   close() {
+    this.state.isolator?.deactivate();
     if (this.props.show) {
-      this.state.isolator?.deactivate();
```

Comment:
Not sure why this behavior was only happening conditionally based on if `this.props.show` is truthy, or if there's a compelling reason to keep it that way, but moving this line out of the conditional fixes the problem in the ticket where aria-hidden wasn't being removed from the top-level app component when `Alert` is closed